multicluster/charts/linkerd-multicluster/templates/gateway.yaml

```diff
@@ -72,44 +24,14 @@ spec:
       annotations:
         {{ include "partials.annotations.created-by" . }}
         linkerd.io/inject: enabled
-        config.linkerd.io/proxy-require-identity-inbound-ports: "{{.Values.gatewayProbePort}},{{.Values.gatewayPort}},4180"
+        config.linkerd.io/proxy-require-identity-inbound-ports: "{{.Values.gatewayProbePort}},{{.Values.gatewayPort}}"
         config.linkerd.io/enable-gateway: "true"
       labels:
         app: {{.Values.gatewayName}}
     spec:
-      volumes:
-        - name: config
-          configMap:
-            name: {{.Values.gatewayName}}-config
-        - name: tmp
-          emptyDir: {}
       containers:
-        - name: nginx
-          readinessProbe:
```

Comment:
Should'nt we add these `readinessProbes` to a locally known proxy endpoint or is it not necessary? 🤔

Reply:
The `pause` container is mostly a placeholder, no need to setup probes, IMO.